index.js

```diff
@@ -26,6 +26,18 @@ if (!rules) {
   );
 }
 
+function getConfig(type) {
+  return {
+    rules: Object.fromEntries(Object.keys(rules).map((rule) => {
```

Comment:
Object.fromEntries is Node 12+

Reply:
Thanks for calling this out! I'll make this a major version change.